features/channel-messaging.yml.dist

```diff
@@ -12,7 +12,7 @@ status:
     firefox: "41"
     firefox_android: "41"
     safari: "5"
-    safari_ios: "4.2"
+    safari_ios: "5"
```

Comment:
this is interesting!

Reply:
I think this is an error in BCD. 

`api.MessagePort` has safari_ios set to `mirror`, and `safari 5` mirrors `safari_ios 4.2`.

`api.MessagePort.worker_support` has `safari_ios` explictly set to `5`, and probably should be `mirror` instead.

This is based on a guess of what seems most likely, not on actually testing functionality or finding release notes.

Reply:
Can't find anything in release notes yet, but filed mdn/browser-compat-data/pull/25407

Reply:
Going to leave this as is and resolve, since it'll be automatically fixed here on rebuild if/when the BCD fix ships.